### PR TITLE
Fix for SA-1 Pack v1.31+

### LINKED ISSUE #19: VWF Dialogues Patch v1.2 is not compatible with SA-1 Pack v1.31.
This patch is not compatible with SA-1 Pack v1.31 and is considered to require an update.
And Lunar Magic 3.01 has the ability to go to over-world in Secondary Entrances, so I cancels the level-out suggestion.

### PR DESCRIPTION
**Pull request by TheLX5**
Fixes #19 

It's SA-1 Hybrid.

And I have no Idea why the whole thing was replaced instead of the relevant lines, it might be related to me using Windows' notepad instead of Notepad++.

### CODE REVIEW

File: patches/vwf_dialogues/vwf_dialogues.asm [hunk 1]
```diff
@@ -1,4665 +1,4676 @@
...
-print "Message register at address $",hex(!message),"."
-print "BG GFX register at address $",hex(!boxbg),"."
-print "BG Color register at address $",hex(!boxcolor),"."
-print "Frame GFX register at address $",hex(!boxframe),"."
-print "Abort Dialogue Processing register at address $",hex(!enddialog),"."
-
-print ""
-print "See Readme for details!"
-print ""
-print "Patch inserted at $",hex(FreecodeStart),",
```
Comment (TheLX5) [inline patches/vwf_dialogues/vwf_dialogues.asm]: Fix #19

**Comment by RPGHacker** [comment]
That's interesting indeed. 0_o
The only thing I could think of here would be line endings, but as far as I know, GitHub (or Git?) should auto-convert line endings to the proper OS, so I don't quite know what could be causing that. Guess this just means I'll have to review the pull request a bit more carefully once I get to it.

**Comment by RPGHacker** [comment]
Oh, okay. There's a "hide whitespace changes" here on GitHub. That makes it a lot easier!

**Comment by ExE-Boss** [comment]
I have opened https://github.com/TheLX5/SMW-Workspace/pull/1 to&nbsp;get the&nbsp;line endings fixed.

**Comment by ExE-Boss** [comment]
You should put
```markdown
Fixes #19
```
into the main PR description (https://github.com/RPGHacker/SMW-Workspace/pull/20#issue-255551714)

**Comment by TheLX5** [comment]
I thought I posted here before... but nope, done btw.

I hope there weren't any bugs related to this fix.

**Comment by RPGHacker** [comment]
Sorry for the super late response.

I couldn't merge this pull request directly, because it seems the patch's structure has changed too much since it was made, but I tried implementing these changes on my own. Could you check out these changes?

https://github.com/RPGHacker/SMW-Workspace/blob/9384aaecf578bfa3653c9109d129a5937a96917c/patches/vwf_dialogues/vwf_dialogues.asm#L441-L448

https://github.com/RPGHacker/SMW-Workspace/blob/9384aaecf578bfa3653c9109d129a5937a96917c/patches/vwf_dialogues/vwf_dialogues.asm#L715-L723

I don't know if this makes any sense - nor do I know what the exact issue was that this PR was trying to fix, or if that issue is even still a thing in SA-1 pack 1.40+, which I used for testing. I didn't really notice any issues that disappeared as a result of me adding those changes, but then I also usually test on a pretty vanilla SMW ROM, so I don't know if that would even cause any of the issues in the first place.

**Comment by RPGHacker** [comment]
Whoops, this is still open. But at this point, I believe I've tested VWF Dialogues 1.30 pretty extensively with SA-1, so I'm gonna assume whatever issue this was meant to address has been fixed by now.

**Pull request closed without merging**